packages/rollup-plugin/src/index.ts

```diff
@@ -104,6 +107,11 @@ export function stylableRollupPlugin({
             }
             extracted.set(id, { css });
 
+            if (dts) {
+                const dtsContent = generateDTSContent({ meta, exports });
+                fs.writeFileSync(id.replace('.st.css', '.d.ts'), dtsContent);
+            }
+
```

Comment:
It should be:
```js
`${id}.d.ts`
```

Reply:
Its that value id not end with `.st.css`

Reply:
Yes, it does. The id is the absolute path.
Therefore we need to extend it with `.d.ts` to tell typescript to look for the declaration file of this stylesheet.